code/xcvm/cosmwasm/tests/build.rs

```diff
@@ -0,0 +1,37 @@
+use sha2::Digest;
+use std::io::Read;
+
+/// Location of the compiled cw20-base contract.
+const CW20_URL: &str =
```

Comment:
We’ve had some issues with nix and build scripts before. @dzmitry-lahoda do you recall why?

Reply:
thank you for tagging me. 

as long as build.rs scripts do not assume anything outside of the source directory they build it is ok.

basically any build which has access to you ssh folder (hahah  - you build and loose access) or any arbitrary http calls (imagine you download some code  and replace part of contract with shit on ci).

that why nix (and any other secure build tool) prevents access to things which where not explicitly described.

so as long as CI builds tests, build.rs is ok.

Reply:
`nix build .#xcvm-tests` to test.

Reply:
@mina86 sorry for many commets, i go some brain hurt :) 

please disable main body of build.rs  in case of NIX_BUILD is defined. 


thank you